credentials/apps/badges/tests/test_management_commands.py

```diff
@@ -26,3 +26,24 @@ def test_handle_with_organization_id(self, mock_credly_api_client):
         call_command("sync_organization_badge_templates", "--organization_id", self.credly_organization.uuid)
         mock_credly_api_client.assert_called_once_with(self.credly_organization.uuid)
         mock_credly_api_client.return_value.sync_organization_badge_templates.assert_called_once_with(1)
+
+
+class TestSyncAccredibleGroupsCommand(TestCase):
+    def setUp(self):
+        self.faker = faker.Faker()
+        self.api_config = AccredibleAPIConfig.objects.create(
+            api_key=self.faker.uuid4(), name=self.faker.word()
+        )
+        AccredibleAPIConfig.objects.bulk_create([AccredibleAPIConfig(api_key=self.faker.uuid4()) for _ in range(5)])
+
+    @mock.patch("credentials.apps.badges.management.commands.sync_accredible_groups.AccredibleAPIClient")
+    def test_handle_no_arguments(self, mock_accredible_api_client):
+        call_command("sync_accredible_groups")
+        self.assertEqual(mock_accredible_api_client.call_count, 6)
```

Comment:
Why exactly 6?
Perhaps this number should be associated with some kind of constant?

Reply:
We created 6 api configs in setup function
```
        # 1
        self.api_config = AccredibleAPIConfig.objects.create(
            api_key=self.faker.uuid4(), name=self.faker.word()
        )

        # +5
        AccredibleAPIConfig.objects.bulk_create([AccredibleAPIConfig(api_key=self.faker.uuid4()) for _ in range(5)])
```

Reply:
Refactored to get count of qs